### DISCUSSION TITLE
apm with signoz/clickhouse

### DISCUSSION THREAD

**Discussion by coutouly** [grafana/grafana#57718] [Dashboards, Panels & Visualizations]
hello,
grafana works well in APM integration with zipkin or jeager or tempo.
I try to do the same for trace panel with clikhouse formating the request like the info in https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/trace.ts#L28
but i didn't succeed.
there is no exlpanation why that doesn't work

do you plan to implement other APM (opentelemetry compatible) to trace panel or nodegraph panel ? other database like elasticsearch or click house ..
is there a doc to do a plugin for that
thanks in advence for your reply

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
